decidim-admin/app/views/decidim/admin/static_page_topics/index.html.erb

```diff
@@ -10,27 +10,48 @@
     </h1>
   </div>
   <% if topics.any? %>
-    <div class="table-scroll">
+    <div class="table-stacked">
       <table class="table-list">
         <thead>
           <tr>
             <th><%= t("activemodel.attributes.static_page_topic.name") %></th>
-            <th></th>
+            <th><%= t("decidim.admin.actions.actions") %></th>
           </tr>
         </thead>
         <tbody>
           <% topics.each do |topic| %>
             <tr>
-              <td>
+              <td data-label="<%= t("activemodel.attributes.static_page_topic.name") %>">
                 <%= link_to translated_attribute(topic.title), [:edit, topic] %>
               </td>
-              <td>
-                <% if allowed_to? :update, :static_page_topic, topic: topic %>
-                  <%= icon_link_to "pencil-line", [:edit, topic], t("actions.edit", scope: "decidim.admin"), class: "action-icon--edit" %>
-                <% end %>
-                <% if allowed_to? :destroy, :static_page_topic, topic: topic %>
-                  <%= icon_link_to "delete-bin-line", topic, t("actions.destroy", scope: "decidim.admin"), class: "action-icon--remove", method: :delete, data: { confirm: t("actions.confirm_destroy", scope: "decidim.admin") } %>
-                <% end %>
+              <td data-label="<%= t("decidim.admin.actions.actions") %>" class="table-list__actions">
+                <button type="button" data-component="dropdown" data-target="actions-topic-<%= topic.id %>" aria-label="<%= t("decidim.admin.actions.actions_label", resource: translated_attribute(topic.title)) %>">
+                  <%= icon "more-fill", class: "text-secondary" %>
+                </button>
+
+                <div class="inline-block relative">
+                  <ul id="actions-topic-<%= topic.id %>" class="dropdown dropdown__action" aria-hidden="true">
+                    <% if allowed_to? :update, :static_page_topic, topic: topic %>
+                      <li class="dropdown__item">
+                        <%= link_to [:edit, topic], class: "dropdown__button" do %>
+                          <%= icon "pencil-line" %>
+                          <%= t("actions.edit", scope: "decidim.admin") %>
+                        <% end %>
+                      </li>
+
+                      <hr>
```

Comment:
This may be problematic if line 34 is true and 45 is false.

Reply:
I think this is just how we were handling the `hr`s until now. This particular case (being able to `update` a Topic but not to `destroy` it) currently doesn't happen, as it is always allowed: 

https://github.com/decidim/decidim/blob/74e8f43c9ca6627a62d8913f920a4f2ac5f60e83/decidim-admin/app/permissions/decidim/admin/permissions.rb#L58